pkg/proxy/iptables/proxier.go

```diff
@@ -169,6 +169,8 @@ type Proxier struct {
 	haveReceivedServiceUpdate   bool // true once we've seen an OnServiceUpdate event
 	haveReceivedEndpointsUpdate bool // true once we've seen an OnEndpointsUpdate event
 	throttle                    flowcontrol.RateLimiter
+	timer                       *time.Timer
+	lastSync                    time.Time
```

Comment:
If we would prefer, I could clean it up and aim for 1.5.1 vs. holding it up?  I don't disagree with @bprashanth 's suggestion.  This isn't really a feature addition at this point.